CHANGELOG.md

```diff
@@ -6,7 +6,8 @@
 
 ##### Enhancements
 
-* None.
+* Update Bazel config to allow targets to be directly consumed. 
```

Comment:
Sorry I know it's an annoyingly arcane syntax, but this needs two trailing spaces for Markdown to translate it to a line break.

```suggestion
* Update Bazel config to allow targets to be directly consumed.  
```